### PR TITLE
52 login signup page patch

### PR DESCRIPTION
**Pull request by Midnite1212**

### CODE REVIEW

File: ui/src/components/MainContainer.js [hunk 1]
```diff
@@ -75,7 +106,7 @@ const MainContainer = () => {
           permissions={["public"]}
           component={ConnectPage}
         />
-        <PrivateRoute
+        {/* <PrivateRoute
```
[the remark this answers is not in the document]
Comment (ksuhartono97) [reply ui/src/components/MainContainer.js:113]: pls double check the containers, a bunch of pages that shouldnt be commented out seem to be missing lol

File: ui/src/components/auth/country.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[
```
Comment (ksuhartono97) [inline ui/src/components/auth/country.json:1]: i dont think this entire file is necessary? you should be using the country list in lists.json

**Review by ksuhartono97 — CHANGES_REQUESTED** [review]
I mostly reviewed the code, the login process seems to work properly.

Signup might still need to be tested but a bunch of these issues should be fixed so we can recheck if the rebase process is fine or not

File: ui/src/components/connect-float/ConnectModal.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
+import {
```
[the remark this answers is not in the document]
Comment (Midnite1212) [reply ui/src/components/connect-float/ConnectModal.js:1]: uh, weird things happen during rebase and this is one of the commits for some reason :") let it be or delete it?

**Review by ksuhartono97 — CHANGES_REQUESTED** [review]

File: ui/src/components/connect-float/ConnectModal.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (ksuhartono97) [reply ui/src/components/connect-float/ConnectModal.js:1]: i think it's fine, this is important to keep

**Review by ksuhartono97 — APPROVED** [review]

**Pull request merged**